portfolio/src/main/webapp/script.js

```diff
@@ -17,85 +17,87 @@ let MAX_COMMENTS = 5;
 
 /** Generate a random greeting */
 function generateRandomGreeting() {
-    //TODO use google translated to get greetings in different languages
-    const greetings =
-        ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  //TODO use google translated to get greetings in different languages
+  const greetings =
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-    // Pick a random greeting.
-    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-    return greeting;
+  // Pick a random greeting.
+  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  return greeting;
 }
 
 /* Update the MAX_COMMENTS */
 function updateNumComments() {
-    const userNumComments = document.querySelector("#numComments").value;
-    getUserComments(userNumComments);
+  const userNumComments = document.querySelector("#numComments").value;
+  getUserComments(userNumComments);
 }
 
 /* Retrieve user comments and display them */
 async function getUserComments(numComments=MAX_COMMENTS) {
-    try {
-        const response = await fetch(`/data?numComments=${numComments}`);
-        const data = await response.json();
+  try {
+    const response = await fetch(`/${await fetchBlobUrl()}?numComments=${numComments}`);
+    const data = await response.json();
 
-        const commentEl = document.querySelector("#user-comments");
-        if(typeof(commentEl) != 'undefined' && commentEl != null) {
-            commentEl.innerText = "";
-            // add a legend
-            commentEl.appendChild(createLegendEl());
+    const commentEl = document.querySelector("#user-comments");
+    if(typeof(commentEl) != 'undefined' && commentEl != null) {
```

Comment:
A cleaner approach would be `if (commentEl) {...}` since any falsy value would fail

Reply:
> A cleaner approach would be `if (commentEl) {...}` since any falsy value would fail

That is right, thank you